src/connection/useRoomConnection.ts

```diff
@@ -33,7 +33,6 @@ export const useRoomConnection = (roomName: string) => {
     // The `onConnect` setting tells it to call the global function under that name when ready.
     sceneElement.setAttribute("networked-scene", {
       adapter: "webrtc",
-      audio: "true",
```

Comment:
#18: you should try this out locally. When this line is added back in, player positions don't sync (on `localhost`)...

Reply:
what the.... and why's that?

Reply:
Haha I would very much like to know. It might be an issue with using WebRTC over localhost, which would explain why it didn't happen on Glitch.